### DISCUSSION TITLE
Standardization Proposal: Adopting American English Spelling in Codebase

### DISCUSSION THREAD

**Discussion by stephanjohnson** [IntentArchitect/Support#27] [General]
I've been doing some deep dives into the source code recently and something that has caught my attention is the variance in English spelling. The generated code output uses a mix of British and American English, and while it's not exactly an issue, it does stand out in the programming world where American English spelling is widely used and recognized as a standard.

The world of programming leans heavily towards American English. Most programming languages were born in the U.S., so the predefined functions, libraries, and classes are typically in American English. Using the same form of spelling can help keep things consistent and prevent any potential misunderstandings.

Having the generated code follow the same standard would, in essence, make it more accessible and easier to read for a larger number of developers worldwide. It would ensure smoother collaboration and potentially encourage contributions from developers who are already familiar with this convention.

Also, remember that most of the documentation, online forums, and coding resources utilize American English. So, by aligning the generated code with American English spelling, we would be in sync with the vast majority of programming literature, making it easier for us to reference and understand external resources.

![image](https://github.com/IntentArchitect/Support/assets/6469385/6dc70228-492f-422d-9f85-e4949031c3fd)

**Comment by dandrejvv** [comment]
Hi @stephanjohnson . This is a fair proposal and one we can discuss first as a team. Will add this to our backlog to look at.
